grazel-gradle-plugin/src/main/kotlin/com/grab/grazel/migrate/target/AndroidInstrumentationBinaryTargetBuilder.kt

```diff
@@ -61,14 +62,16 @@ internal class AndroidInstrumentationBinaryTargetBuilder
                 matchedVariant = matchedVariant,
                 sourceSetType = SourceSetType.JAVA_KOTLIN,
             )
-            add(androidInstrumentationBinData.toTarget())
+            add(androidInstrumentationBinData.toTarget(project))
```

Comment:
Can we move this to extractor instead of directly mapping the value here? 

Later we can make extractors cacheable and source of truth for migration data. 

So `androidInstrumentationBinData` itself will contain compose field

Reply:
Moved it to extractor 🙇🏻‍♂️